Library/Formula/cairo.rb

```diff
@@ -1,10 +1,9 @@
-require "formula"
-
 class Cairo < Formula
   homepage "http://cairographics.org/"
   url "http://cairographics.org/releases/cairo-1.14.2.tar.xz"
   mirror "http://www.mirrorservice.org/sites/ftp.netbsd.org/pub/pkgsrc/distfiles/cairo-1.14.2.tar.xz"
```

Comment:
It'd suggest `https` for this mirror, it supports it:
https://www.mirrorservice.org/sites/ftp.netbsd.org/pub/pkgsrc/distfiles/cairo-1.14.2.tar.xz